util/json.go.tmpl

```diff
@@ -0,0 +1,57 @@
+{{/*
+    JSON Converter CC (JSONify)
+    
+    Usage: run the json command to find out :)
+
+    Recommended Trigger Type: Command
+    Recommended Trigger: json
+    
+    Credits:
+    Devonte <https://github.com/devnote-dev>
+*/}}
+
+{{$a := parseArgs 2 "```json <Channel:Mention> <Message:ID>\njson <Channel:Name> <Message:ID>\njson <Channel:ID> <Message:ID>\njson 0 <Message:ID>\n```**Optional Flags:**\n```\n[-j strict-format:flag]\n[-file/f attachment:flag]```"
+(carg "string" "chan") (carg "int" "msgID") (carg "string" "flag: -j | -f")}}
+{{$type := dict 0 "Default Message" 6 "Pinned Message" 7 "Join Message" 8 "Boost Message" 9 "Teir 1 Boost Message" 10 "Teir 2 Boost Message" 11 "Teir 3 Boost Message" 12 "Followed Channel Message" 19 "Reply Message"}}
+{{$struct := "Unknown"}}{{$ver := "JSONify v4.15"}}{{$fa := false}}{{$chan := .Channel}}
+{{$link := joinStr "/" "https://discordapp.com/channels" .Guild.ID .Channel.ID ($a.Get 1)}}{{$ctx := ""}}
+
+{{$l := sendMessageRetID nil (cembed "description" "Converting Message... <a:loading:760219029620523008>")}}
+{{if ne ($a.Get 0) "0"}}
+    {{$chan = getChannel (or (reFind `\d+` ($a.Get 0)) ($a.Get 0))}}
```

Comment:
This should check if the `$chan` input is valid, otherwise the code will error out.
![image](https://user-images.githubusercontent.com/3849946/114313980-2f747080-9acf-11eb-8e81-c2971ebe2c32.png)

Also, I think `nil` should be accepted as an input, as it is commonly used on CC's.

Reply:
The commit states that `0` is the placeholder for the triggering channel, not `nil` literally.

Reply:
I know. That's why I said: `Also, I think nil should be accepted as an input`. This is just a suggestion.
But, the code erroring out like this is not good at all. It will error out if any invalid channel is provided. `nil` was just an example.
![image](https://user-images.githubusercontent.com/3849946/114319123-7cb00c80-9ae6-11eb-84af-cf67d8fb365f.png)
This error should be handled just like the invalid message error is handled. 
![image](https://user-images.githubusercontent.com/3849946/114319196-cb5da680-9ae6-11eb-8f28-6984a80dba5c.png)

Reply:
Fixed in the most recent commit.